app.vue

```diff
@@ -44,6 +44,20 @@ useHead({
       async: true,
       type: 'text/javascript',
     },
+    {
+      innerHTML: `
+        (function(h,o,t,j,a,r){
+        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+        h._hjSettings={hjid:3333046,hjsv:6};
+        a=o.getElementsByTagName('head')[0];
+        r=o.createElement('script');r.async=1;
+        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+        a.appendChild(r);
+        })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
+      `,
+      type: 'text/javascript',
+      charset: 'utf-8',
```

Comment:
I did this consistently with how Freshdesk was done.

I think you can also define it as a plugin like:
```// nuxt.config.js
{
  // ...
  plugins: ['~/plugins/hotjar']
}```

and then add the implementation code (without the script tags) to plugins/hotjar.js like so:
```
// plugins/hotjar.js
(function(h,o,t,j,a,r){...})(...);```